src/overcooked_ai_py/data/testing/test_state_visualizer/Create_state_visualizer_tests.ipynb

```diff
@@ -0,0 +1,641 @@
+{
```

Comment:
Just to clarify (this might be a dumb question) – is this test notebook automatically run and checked for errors? Or is the "test" manual, as in one would have to open the notebook and run it to check?

I realize that you have `visualization_test.py` that should do testing otherwise – was mainly asking out of curiousity.

Reply:
The notebook is for manually making changes in test cases used in `visualization_test.py` automatically. When you change something that changes visual representation of the test cases (you can check this by running `visualization_test.py` you need to run the cells one by one manually checking if output matches the expectactions.

Reply:
Ah I see – makes sense! Thanks for clarifying!